yeonjae/programmers-1/서울에서김서방찾기.js

```diff
@@ -0,0 +1,17 @@
+function solution(seoul) {
+  for (const el of seoul) {
+    const idx = seoul.indexOf(el);
+
+    if (el === "Kim") {
+      return `김서방은 ${idx}에 있다`;
+    }
+  }
+}
```

Comment:
```suggestion
function solution(seoul) {
  const kimIdx = seoul.indexOf("Kim");
  
  return `김서방은 ${idx}에 있다`;
}
```
- `indexOf` 내장 메서드는 배열을 순회하면서 인자로 들어가는 요소를 찾으면 인덱스를 반환해주기 때문에 굳이 for문을 통해서 반복할 필요가 없습니다 👍

Reply:
지금보니까 왜 for문을 돌렸는지...! ㅎㅎ  앞으로 좀 더 깊게 생각하면서 풀겠습니당 :)